cumulus/ansible/tasks/playbooks/volumes/ec2/attach.yml

```diff
@@ -84,6 +83,7 @@
               path: "volumes/{{ girder_volume_id }}/clusters/{{ girder_cluster_id }}/attach/complete"
               data:
                 path: "{{ path }}"
+                status: "{{ vol.volume.status }}"
           delegate_to: localhost
```

Comment:
@kotfic Not sure what you think about this change. The issue is that if we set the status to 'in-use' before attach/complete is he then we can try to detach a volume that is in the process of being attach ( which is what the test case was doing ). Also looking at the change, I'm not sure we need to pass in the status to attach/complete.